corecomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2020/ui/ChromeCustomTabsNavigator.kt

```diff
@@ -38,7 +39,11 @@ class ChromeCustomTabsNavigator(private val context: Context) : Navigator<Chrome
             .setToolbarColor(context.getThemeColor(R.attr.colorAccent))
 
         val intent = builder.build()
-        intent.launchUrl(context, Uri.parse(url))
+        try {
+            intent.launchUrl(context, Uri.parse(url))
+        } catch (e: ActivityNotFoundException) {
+        } catch (e: Exception) {
```

Comment:
I think it is not needed. When there really is a problem, we don't know the problem. 🙏